### ISSUE TITLE
says token is invalid when it is idk try to check it out ig

### ISSUE THREAD

**Issue by AstroOnYT** [AstraaDev/Discord-SelfBot#1]

**Comment by AstraaDev** [comment]
Your token is probably invalid because its validity is tested each time the script is run. Try another one and if it still doesn't work, contact me on discord: https://dsc.gg/astraadev

**Comment by AstroOnYT** [comment]
bro i used ur token grabber and use console on 2 different accounts so i think its the script but idk

**Comment by AstraaDev** [comment]
Try to connect to the account with my multi tool (https://github.com/AstraaDev/Discord-All-Tools-In-One). If it works, I'll try to find the problem, if not, it's because of your token.

**Comment by AstroOnYT** [comment]
yep it definitly does work when i use auto login with my token on the multi tool but for some reason is thinks my token is invalid for this selfbot

**Comment by AstraaDev** [comment]
Fuck, can you create a ticket on my server so we can see where the problem comes from?

**Comment by FuckingToasters** [comment]
> Fuck, can you create a ticket on my server so we can see where the problem comes from?

Tell him to enable all intents :)

**Comment by DanielTheGeek** [comment]
The issue could possibly be from the requirements.txt file. Latest versions of all required packages would install at the time of running the script. This means the version of each dependency @AstraaDev used while testing would most likely be different from what others are using 3 months down the line. I'd suggest @AstraaDev updates requirements.txt to include specific versions for each dependency. For instance the new Discord.py requires intents before calling discord.Client().

**Comment by SlepPoweredBy2** [comment]
this no work no mo :(

**Comment by SlepPoweredBy2** [comment]
Btw this is a self bot but do I need to use a bot in my server or my own token because this doesn't work
